invokeai/app/invocations/grounding_dino.py

```diff
@@ -0,0 +1,95 @@
+from pathlib import Path
+
+import torch
+from PIL import Image
+from transformers import pipeline
+from transformers.pipelines import ZeroShotObjectDetectionPipeline
+
+from invokeai.app.invocations.baseinvocation import BaseInvocation, invocation
+from invokeai.app.invocations.fields import BoundingBoxField, ImageField, InputField
+from invokeai.app.invocations.primitives import BoundingBoxCollectionOutput
+from invokeai.app.services.shared.invocation_context import InvocationContext
+from invokeai.backend.image_util.grounding_dino.detection_result import DetectionResult
+from invokeai.backend.image_util.grounding_dino.grounding_dino_pipeline import GroundingDinoPipeline
+
+GROUNDING_DINO_MODEL_ID = "IDEA-Research/grounding-dino-tiny"
+
+
+@invocation(
+    "grounding_dino",
+    title="Grounding DINO (Text Prompt Object Detection)",
+    tags=["prompt", "object detection"],
+    category="image",
+    version="1.0.0",
+)
+class GroundingDinoInvocation(BaseInvocation):
+    """Runs a Grounding DINO model (https://arxiv.org/pdf/2303.05499). Performs zero-shot bounding-box object detection
```

Comment:
I understand why this info is written here but I think a lot of it needs to be put side in the comment and let the class description be precise because it also double functions as node info in the UI. Think this part needs to be kept clean and precise.

![opera_40tAm63u2L](https://github.com/user-attachments/assets/18101176-88b7-4664-992e-a8b6d51d6996)

Reply:
Shortened in [c6d49e8](https://github.com/invoke-ai/InvokeAI/pull/6701/commits/c6d49e8b1fc2a4e1d41c128e3a5123719424b581)